awx/ui_next/src/screens/Job/Job.jsx

```diff
@@ -139,6 +146,19 @@ class Job extends Component {
                 path="/jobs/:type/:id/output"
                 render={() => <JobOutput type={match.params.type} job={job} />}
               />,
+              <Route
```

Comment:
This scenario seems more like there's no matching route for the url and less like there was an attempt to access a resource that wasn't found.

If the URL pattern is bad, there's no need to try and represent it as we would a 404 (IMHO). For unmatched routes, I think it would simplify things quite a bit if we handled those generically at the top-level of the router.

Reply:
The problem is the top-level router has already matched it to route beginning `/jobs` and rendered this component. So we need to catch a more deeply-nested bad url here.

Reply:
Ah, yep.  I see that now, thank you. :+1: